### PR TITLE
Consolidated some of the config/option creation

### PR DESCRIPTION
**Pull request by anthonysterling**
I've just added some internal methods to handle the casting and formatting of the
configuration/option creation.

### CODE REVIEW

File: src/Configurations/PageRulesActions.php [hunk 1]
```diff
@@ -54,29 +46,23 @@ public function setBypassCacheOnCookie(bool $value)
             throw new ConfigurationsException("Invalid cookie string.");
         }
 
-        $object = new \stdClass();
-        $object->id = "bypass_cache_on_cookie";
-        $object->value = $value;
```
Comment (anthonysterling) [inline src/Configurations/PageRulesActions.php]: This is a `bool` but get passed in raw unlike the on/off conversation, is this correct?

File: src/Configurations/PageRulesActions.php [hunk 2]
```diff
@@ -98,38 +82,30 @@ public function setCacheOnCookie(bool $value)
             throw new ConfigurationsException("Invalid cookie string.");
         }
 
-        $object = new \stdClass();
-        $object->id = "cache_on_cookie";
-        $object->value = $value;
```
Comment (anthonysterling) [inline src/Configurations/PageRulesActions.php]: This is a `bool` but get passed in raw unlike the on/off conversation, is this correct?

File: src/Configurations/PageRulesActions.php [hunk 3]
```diff
@@ -138,135 +114,107 @@ public function setEdgeCacheTTL(int $value)
...
-
-        array_push($this->configs, $object);
+        $this->addConfigurationOption("disable_security", [
+            'value' => $this->getBoolAsOnOrOff($active)
+        ]);
     }
 
     public function setForwardingURL(int $statusCode, string $forwardingUrl)
     {
-        if (in_array($statusCode, ['301', '302'])) {
```
Comment (anthonysterling) [inline src/Configurations/PageRulesActions.php]: I've inverted this as I think this is incorrect?

File: src/Configurations/PageRulesActions.php (hunk 1 above)
Comment (IcyApril) [reply src/Configurations/PageRulesActions.php]: There's a slight inaccuracy I'll adjust here in a separate PR, in essence the value tied to Bypass Cache on Cookie (or Cache on Cookie) for that matter should be a string.

**Pull request merged**